README.md

```diff
@@ -874,15 +874,7 @@ easier understanding and reading of a test.
     end
     ```
 
-* Add an example ensuring that the FactoryGirl.created model is valid.
```

Comment:
I would agree that this should be removed or the verbiage should be changed. If the context is to verify that the model is valid in the model spec, we shouldn't be testing a mock (based on the first bullet point under Models).